sunbird_service_api_test/src/test/resources/templates/organisation/read/testReadOrgSuccessWithValidProviderAndExternalId/request.json

```diff
@@ -1,5 +1,6 @@
 {
   "request": {
+    "organisationId": "${organisationId}",
```

Comment:
Why we always send     "organisationId": "${organisationId}" is added in request.